src/website/app/pages/Typography/examples.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '../../../../utils';
+import Heading from '../../Heading';
+import Paragraph from '../../Paragraph';
+
+type TypeExample = {
+  heading: MnrlReactNode,
+  description: MnrlReactNode,
+  examples: Array<Object>
+};
+
+type TypeExamples = Array<TypeExample>;
+
+const styles = {
+  description: ({ theme }) => ({
+    margin: `0 0 ${theme.spacing_quad} 0`
+  }),
+  list: ({ theme }) => ({
+    listStyle: 'square',
+    paddingLeft: theme.spacing_triple
+  })
+};
+
+const List = createStyledComponent('ul', styles.list);
+const Description = createStyledComponent(Paragraph, styles.description);
+
+const examples: TypeExamples = [
+  {
+    heading: (
+      <Heading level={2} id="long-form-text">
+        Long-form Text
+      </Heading>
+    ),
+    description: (
+      <Description variant="prose">
+        Default text and caption styles are intended for sections of content
+        that provide description and assistance. {' '}
+        <strong>Paragraph text</strong> is primarily used for content sections.{' '}
+        <strong>Caption Text</strong> is used to highlight or offset content,
+        such as inline help or when captioning of dashboard content.
```

Comment:
"captioning of dashboard content" — Remove "of".

Reply:
addressed